From ef42f9856344b360c7be378766ace69041e9a67a Mon Sep 17 00:00:00 2001
From: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Date: Wed, 8 Mar 2017 13:28:06 +0900
Subject: [PATCH 567/909] arm64: dts: r8a7796: Add DEVFREQ support for GSX

commit 1b6f0c642729a5811f287284e66259cb0b3e65ae from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

This patch supports GSX frequency scaling using devfreq.

Signed-off-by: Gaku Inami <gaku.inami.xw@bp.renesas.com>
Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/renesas/r8a7796.dtsi | 21 +++++++++++++++++++++
 1 file changed, 21 insertions(+)

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index 2cc5dcaa6075..c50d134b52cf 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -581,6 +581,27 @@
 		clock-frequency = <0>;
 	};
 
+	gsx_opp_table: gsx_opp_table {
+		compatible = "operating-points-v2";
+
+		opp-200000000 {
+			opp-hz = /bits/ 64 <200000000>;
+			opp-microvolt = <830000>;
+		};
+		opp-300000000 {
+			opp-hz = /bits/ 64 <300000000>;
+			opp-microvolt = <830000>;
+		};
+		opp-400000000 {
+			opp-hz = /bits/ 64 <400000000>;
+			opp-microvolt = <830000>;
+		};
+		opp-600000000 {
+			opp-hz = /bits/ 64 <600000000>;
+			opp-microvolt = <830000>;
+		};
+	};
+
 	/* MSIOF reference clock - to be overridden by boards that provide it */
 	msiof_ref_clk: msiof-ref-clock {
 		compatible = "fixed-clock";
-- 
2.17.1

